src/bindings/js/node/src/tensor.cpp

```diff
@@ -66,6 +66,12 @@ Napi::Object TensorWrap::wrap(Napi::Env env, ov::Tensor tensor) {
 }
 
 Napi::Value TensorWrap::get_data(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "getData() does not accept any arguments.");
+        return env.Undefind
```

Comment:
```suggestion
        return env.Undefined();
```

Reply:
Hi @almilosz, thank you for pointing out the issue. Sorry, I forgot to push the commit. Doing the push right now. Can you please review it.